README.md

```diff
@@ -122,145 +162,129 @@ firmwares/
               *.ipk
 ```
 
-As you notice there are several different image variants ("backbone", "default", etc.).
-These different *packages lists* are defined in `packages/`.
-See the "Features" section above for a description of the purpose of each package list.
-With the "OpenWrt-Imagebuilder" you can assemble your own image variant with your
-*packages lists* without having to compile everything yourself. The "OpenWrt-SDK" is
-the fastest way to build your own packages or programs without compiling OpenWrt itself.
-The "initrd" directory contains some initrd-images for netboot, which are required on
-some boards to initially install OpenWrt.
+As you notice, there are several different image variants ("backbone", "default", etc.).
+These different *package lists* are defined in `packages/`.
```

Comment:
can you also change "pacakges/" to "packagelists/"? this was missed on previous updates.